renderdoc/driver/gl/gl_replay.cpp

```diff
@@ -3205,7 +3205,20 @@ ShaderDebugTrace GLReplay::DebugThread(uint32_t eventID, uint32_t groupid[3], ui
 
 const GLHookSet &GetRealGLFunctions();
 
+#if defined(RENDERDOC_SUPPORT_GL)
```

Comment:
You'll need to add this define (`RENDERDOC_SUPPORT_GL`) in the VS project so that this continues to work on windows, as currently it's only set through cmake.

If you don't have VS set up let me know, I can just make that change when merging.